budgetApp1/budgetApp1/AccountViewController.swift

```diff
@@ -11,91 +11,61 @@ class AccountViewController: UIViewController {
 
     // MARK: - variables
     @IBOutlet private var tableView: UITableView!
+    @IBOutlet weak var accountHeaderLabel: UILabel!
     private lazy var viewModel = AccountViewModel(repository: AccountRepository(), delegate: self)
     
     // MARK: - UISpecs Dependency
     private let uiSpecs = UISpecs()
     
     // MARK: - functions
+    private func setupTableView() {
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(AccountViewCell.nib(), forCellReuseIdentifier: NibIdentifiers.accountViewCellIdentifier)
+        tableView.separatorStyle = .singleLine
+        tableView.separatorColor = uiSpecs.primaryColourOne
+        tableView.separatorInset = UIEdgeInsets(top: 0, left: 16, bottom: 0, right: 16)
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        accountHeaderLabel.text = "Accounts"
         setupTableView()
         viewModel.fetchAccounts()
     }
     
-    private func setupTableView() {
-        tableView.delegate = self
-        tableView.dataSource = self
-        tableView.register(AccountViewCell.nib(), forCellReuseIdentifier: NibIdentifiers.accountViewCellIdentifier)
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        if let selectedIndexPath = tableView.indexPathForSelectedRow {
+            tableView.deselectRow(at: selectedIndexPath, animated: animated)
+        }
     }
 }
 
 // MARK: - TableView Delegate
 extension AccountViewController: UITableViewDelegate, UITableViewDataSource {
     
     func numberOfSections(in tableView: UITableView) -> Int {
-        viewModel.accountListCount
+        1
```

Comment:
why is this hardcoded to 1?
please do something similar to what you had like viewModel.accountListCount

Reply:
I just removed it because I do not need it.